CONTRIBUTING.md

```diff
@@ -2,52 +2,75 @@
 
-We welcome feedback with or without pull requests. If you have an idea for how
-to improve the project, great! All we ask is that you take the time to write a
-clear and concise explanation of what need you are trying to solve. If you have
-thoughts on _how_ it can be solved, include those too!
+We welcome feedback with or without pull requests. If you have an idea for how to improve the project, great! All we ask is that you take the time to write a clear and concise explanation of the need you are trying to solve. If you have thoughts on _how_ it can be solved, include those too!
+
+To submit a feature request, follow these steps:
+
+1. Click the [*New*][5] button from the [*Issues*][1] page. A new interface will appear.
+2. Find _Feature request_ on the list, and click [_Get Started_][7]. A new interface will appear.
+3. Add a title for your feature request. It should briefly describe your requested feature.
+4. Follow the template that appears in the Write text box. This is the best way to explain your request. Be clear and concise in your responses.
+* **Is your feature request related to a problem? Please describe.** - Describe what the problem is. For example, “I'm always frustrated when [...]”.
+* **Describe the solution you'd like** - Describe what you want to happen.
+* **Describe alternatives you've considered** - Describe any alternative solutions or features you've considered.
+* **Additional context** - You can add any other context or screenshots about the feature request in this section.
+5. Click _Submit new issue_ to submit the feature request.
 
 The best way to see a feature added, however, is to submit a pull request.
 
 ## Pull Requests
 
-* Before creating your pull request, it's usually worth asking if the code
-  you're planning on writing will actually be considered for merging. You can
-  do this by [opening an issue][1] and asking. It may also help give the
-  maintainers context for when the time comes to review your code.
+* Before creating your pull request, it's usually worth asking whether the code you're planning on writing will be considered for merging. You can do this by [opening an issue][1] and asking. It may also help give the maintainers context for when the time comes to review your code.
 
-* Ensure your [commit messages are well-written][2]. This can double as your
-  pull request message, so it pays to take the time to write a clear message.
+* Ensure your [commit messages are well-written][2]. This can double as your pull request message, so it pays to take the time to write a clear message.
 
-* Add tests for your feature. You should be able to look at other tests for
-  examples. If you're unsure, don't hesitate to [open an issue][1] and ask!
+* Add tests for your feature. You should be able to [look at other tests][4] for examples. If you're unsure, don't hesitate to [open an issue][1] and ask!
 
 * Submit your pull request!
 
+To submit your pull request, follow these steps:
```

Comment:
This is not Rosetta specific. This is github specific process. We would rather link to a github doc page rather than writing our own guide. Like we can link [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

Reply:
Updated in newest commit.